source/parser/block.civet

```diff
@@ -221,11 +225,29 @@ function needsPrecedingSemicolon(exp: ASTNode)
       if exp.children
         needsPrecedingSemicolon exp.children
 
+/** Is this node a top-level statement in its block? i.e. not an expression */
+function isStatementInBlock(exp: ASTNodeObject)
```

Comment:
imo this function name is a little misleading because it doesn't return a boolean indicating whether it is -- instead, it returns `undefined` if not, and an object containing the block and a couple other things if so.